### DISCUSSION TITLE
node on mini pc

### DISCUSSION THREAD

**Discussion by ghost** [veryengineer/collab#5] [General]
Hey,
I am Max L and new on github and i'm deffo no coder. i run nodes for 1.5 year now. started with one ended up with 2. now my 2nd node was odl and want to replace it. i found the perfect solution for me and don't cost much. It's a hp prodesk 400 g3 mini with i3-7100t, 2x 4gb ddr4, 128gb ssd. and came with windows 10 pro. i bought it used for 100 euro.

again i am new to github and dont know anything about it, only that it is for coders. and i am a dislect so this not my piece of cake. i dont understand the other discussions because i am a carpenter and basicly know nothing about the pc language you guys speak.
much love Max L.

**Comment by patricklodder** [comment]
Hey Max,

That's awesome! How's your new node performing?

PS: Don't be afraid to ask questions if something is unclear, it is helpful not only for you but also for whom you're asking.

**Reply by ghost** [reply]
hey, i got my mini node running for 24 hours now and it doing like it should. only i had to put it on port 22557 because i run other node on 22556. On 22557 i have way fewer connections. end up putting some peers in manual. doesn't seem to find many peers compared to 22556. any specific specs i can send here to show performens?

**Reply by patricklodder** [reply]
>  i had to put it on port 22557 because i run other node on 22556

Running on a non-standard port will make your node incompliant with the DNS seeders (as do tor-only nodes). I will tune my network analyzing software to get better insights into what ports are used - you're not the only operator doing this, but it would be good to know how widespread this solution is. Just realize that the node with the non-standard port is the one that will get the least traffic - you can plan for that.

> any specific specs i can send here to show performens?

We have some benchmarking tools: 
- `bench_dogecoin` that measures raw throughput of important low-level functions - I can compile this for Windows for you
- `-debug=bench` as a startup argument to give performance statistics on important operations in debug.log (this will make debug.log grow fast, so I'd recommend only running this temporarily)

**Reply by ghost** [reply]
hey, i did benchmarkwith geekbench, i did one with the node running "https://browser.geekbench.com/v6/cpu/163371". and one without the node running "https://browser.geekbench.com/v6/cpu/163509" idk if it any help. the bench_dogecoin i can't find that. and try'd those command's in the debug and i get "Method not found (code -32601)".

**Reply by patricklodder** [reply]
You can find working bench binaries here: https://github.com/veryengineer/collab/discussions/8#discussioncomment-4863054
